src/components/User/Limits/Limits.Reviewer.vue

```diff
@@ -356,8 +359,8 @@ export default {
       return this.desiredLimitDetails.documents
     },
 
-    requestType () {
-      return LIMITS_REQUEST_STATES_STR[get(this.desiredLimitDetails, 'requestType')]
+    localizedRequestType () {
+      return LIMITS_REQUEST_STATES_STR[get(this.desiredLimitDetails, 'localizedRequestType')]
```

Comment:
Remove intermediary.
I think you can remove LIMITS_REQUEST_STATES_STR const

Reply:
done